CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added Source Code Badge
```

Comment:
Can you update this to be a bit more descriptive?

Reply:
Changed here : https://github.com/cerner/terra-dev-site/pull/243/commits/6dde4584e4c4d54343e4b4f2efdf3f0be133ebf7